### DISCUSSION TITLE
add new row in multiple database table from a single model

### DISCUSSION THREAD

**Discussion by lovelyarora24** [Laravel-Backpack/community-forum#561] [Q&A (Help)] [answered]
I have a location table which contains some fields and I have created local crud in laravel backup. Now I want to add three entries in different database tables from same location model. 

i.e  Table: **Location**  

CRUD already created for this table.
Location is adding and saving through backpack admin UI 
loc_id
name
lat_long
address

Table: **partners**:
partner_id 
loc_id (foreign key of location table)
lat_long

Table: **courier**
courier_id
loc_id (foreign key of location table)
lat_long
address

Can you please guide how can I do this?

**Comment by christmex** [comment, marked as answer]
Hi @lovelyarora24 I ever asked this question before, about insert to another table from 1 form, they suggested me to use "model event" or "extend the insert operation", check this out, maybe the best one is to use model event 
https://backpackforlaravel.com/docs/5.x/release-notes#use-model-events-inside-crudcontrollers

also, I have another opinion, maybe I need to make my own custom operation to handle this unique case, this is the best for me, so far
https://backpackforlaravel.com/docs/5.x/crud-operations#creating-a-custom-operation-1

this is the example of how we do that custom operation but still use the great power field from backpack
https://backpackforlaravel.com/articles/tutorials/how-to-create-a-custom-operation-that-uses-backpack-fields


Sorry for my English, I hope u found it useful, let me know If you have another question mate, cheers
